pages/common/mysql.md

```diff
@@ -17,3 +17,7 @@
 - Execute SQL statements in a script file (batch file):
 
 `mysql {{database_name}} < {{script.sql}}`
+
+- Connect to a database through a Unix socket:
```

Comment:
Let us shift this one example above. All "connecting" examples should be together for better readability.

Reply:
Ah sure that makes sense. I'll do that :)
Thank you for your review.